dev-workflow-ui/webContent/includes/template/notifications.xhtml

```diff
@@ -37,7 +37,7 @@
         <p:dataScroller id="notifications" lazy="true" styleClass="notification-scroll" value="#{notificationBean.dataModel}" var="notification" chunkSize="20" mode="inline">
           <div class="#{notification.style} notification-container">
             <div class="p-d-flex p-jc-between">
-              <h:outputText id="notificationMessage" value="#{notification.message}" />
+              <h:outputText id="notificationMessage" value="#{notification.message}" escape="false"/>
```

Comment:
We need to be sure that data can not be manipulated otherwise someone could make an XSS attack.

Let's say we have the task name in the message. If the task name comes from user input and we display it here, than we would have an XSS hole?

I recommend to manually strip javascript out of the message and maybe other stuff? We have such libaries in the core.

Reply:
Yes, right. Was also thinking about that. Doesn't ivy.cm.co already escape <script> tags?

Reply:
I can't remember that ivy.cm.co is doing that.